cardano-cli/src/Cardano/CLI/Types/Errors/GovernanceCmdError.hs

```diff
@@ -53,6 +56,10 @@ data GovernanceCmdError
   | -- Legacy - remove me after cardano-cli transitions to new era based structure
     GovernanceCmdMIRCertNotSupportedInConway
   | GovernanceCmdGenesisDelegationNotSupportedInConway
+  | GovernanceCmdMismatchedDRepMetadataHashError
+      !(L.SafeHash L.StandardCrypto L.AnchorData)
+      !(L.SafeHash L.StandardCrypto L.AnchorData)
```

Comment:
Can you add a comments which one is expected and which one is actual?

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/commit/4d736e59bd7f062fb99da1691dfa05dd1b210c80